cpp/ql/lib/semmle/code/cpp/controlflow/IRGuards.qll

```diff
@@ -236,6 +265,17 @@ private class GuardConditionFromIR extends GuardCondition {
     )
   }
 
+  /**
+   * Holds if (determined by this guard) `e < k` evaluates to `isLessThan` if
+   * this expression evaluates to `value`.
+   */
```

Comment:
QLDoc seems redundant, because it's an `override`?

Reply:
Indeed! I think I copied it from somewhere else (before editing it), and it seems like we've made this mistake in other places in this file. I'll remove all the redundant ones!

Reply:
Fixed all of them in 3a7b80da47417183f73fe68b67ba0c647ab65c8f